src/rgw/rgw_lc.cc

```diff
@@ -539,6 +539,36 @@ static bool pass_size_limit_checks(const DoutPrefixProvider *dpp, lc_op_ctx& oc)
 static std::string lc_id = "rgw lifecycle";
 static std::string lc_req_id = "0";
 
+static void may_be_send_notification(const DoutPrefixProvider *dpp,
```

Comment:
nit: I think we can change to `send_notification()` sending is not conditional, just could fail.
the fact that it does not return the error code mean that the failure should be ignored.

Reply:
done